projects/data_generation/synthetic-multi-view-facial-image-generation/requirements.txt

```diff
@@ -0,0 +1,11 @@
+torch>=1.0.0
+torchvision
+dominate>=2.3.1
+dill
+scikit-image
+numpy>=1.15.4
+scipy>=1.1.0
+matplotlib==3.0.2
+opencv-python>=3.4.3.18
+tensorboard==1.14.0
+face-alignment==1.0.0
```

Comment:
Please make sure that the dependencies are compatible with the toolkit dependencies.
For example:
- `matplot` version used by the library is currently version 2.2.2
- `opencv-python` is 4.5.1.48
- `tensorboard` is 2.4.1

Reply:
the minimum requrements are satified

Reply:
For `tensorboard` this code is still using a different version than the OpenDR toolkit.